libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -815,21 +815,18 @@ def _insert_embeddings(
             raise
         return ids
 
-    def delete(self, ids: Optional[List[str]] = None, **kwargs: Any) -> Optional[bool]:
```

Comment:
as per the documentation at https://python.langchain.com/api_reference/core/vectorstores/langchain_core.vectorstores.base.VectorStore.html#langchain_core.vectorstores.base.VectorStore.delete , isn't the return type `Optional[bool]`?